src/components/footer.jsx

```diff
@@ -79,7 +79,7 @@ function Footer() {
             )}
           />
           <Grid
-            className="usa-footer__contact-links footer"
+            className="usa-footer__contact-links footer" // TODO: contact-links?
```

Comment:
I updated the classname here in my most recent PR. 

Also, what do you think about putting to dos in airtable instead of as comments?

Reply:
I put the TODOS in the Airtable but forgot that I included this here. I'll remove it.